### ISSUE TITLE
Able to add duplicate file when avoidRepeatedFiles is set

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bdshadow/gwtupload#156]
```
What steps will reproduce the problem?

1. Chose file to upload
2. Chose the same file to upload, before status "Done" is set for (1). 
3. Have fun with surprises :)

What is the expected output? What do you see instead?

Should be impossible to add the same file if avoidRepeatedFiles is set.

What version of the product are you using? On what operating system?

0.6.4

Please provide any additional information below.

The problem is in onFileInputChanged checks only fileDone list.
```

Original issue reported on code.google.com by `vpche...@productengine.com` on 16 Jul 2012 at 1:21

**Comment by GoogleCodeExporter** [comment]
Original comment by `manuel.carrasco.m` on 3 Dec 2012 at 8:34
- Changed state: **Accepted**
- Added labels: **Milestone-0.6.6**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision e6c4b25db9f9.
```

Original comment by `manuel.carrasco.m` on 3 Dec 2012 at 8:35
- Changed state: **Fixed**
